packages/themes/doete/tokens/light/primitive/color.json

```diff
@@ -104,6 +104,19 @@
         "800": { "$type": "color", "$value": "#584915" },
         "900": { "$type": "color", "$value": "#3f3308" },
         "950": { "$type": "color", "$value": "#261f00" }
+      },
+      "base": {
+        "emerald": { "$type": "color", "$value": "#004D44" },
```

Comment:
Let's remove from the `primitive/color`:
- emerald
- gold
- purple
Because they are more brand-specific.